scripts/qa_crowbarsetup.sh

```diff
@@ -1298,48 +1303,111 @@ function create_repos_yml_for_platform()
     for repo in $*; do
         repo_name=${repo%%=*}
         repo_url=${repo##*=}
-        if [ -d "$tftpboot_dir/$repo_name" ]; then
-            if [ -z "$platform_created" ]; then
-                echo "$platform:"
-                echo "  $arch:"
-                platform_created=1
-            fi
-
-            echo "    $repo_name:"
-            echo "      url: '$repo_url'"
+        if [ -z "$platform_created" ]; then
+            echo "$platform:"
+            echo "  $arch:"
+            platform_created=1
         fi
+
+        echo "    $repo_name:"
+        echo "      url: '$repo_url'"
     done
 }
 
 function create_repos_yml()
 {
     local repos_yml="/etc/crowbar/repos.yml"
     local tmp_yml=$(mktemp).yml
+    local additional_repos=
 
     echo --- > $tmp_yml
 
     if iscloudver 6; then
+        additional_repos=
```

Comment:
It is. But I see more value in making the code consistent in all cases, because sometimes we need to reset the variable. That way, if someone does some copy & paste, they won't forget to reset the variable.